package.json

```diff
@@ -139,6 +139,9 @@
     "test-client": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=client test/runner.js",
     "test-server": "NODE_ENV=test NODE_PATH=test:server:client TEST_ROOT=server test/runner.js",
     "test-test": "NODE_ENV=test NODE_PATH=test:client TEST_ROOT=test test/runner.js",
+    "test-client:watch": "nodemon -e js,jsx --exec npm run test-client",
```

Comment:
Is `nodemon` smart enough to ignore js/jsx files in `node_modules`? I could also see `build/` or `public/` generated scripts triggering this unnecessarily, but probably not a huge issue (hot reloading doesn't save to disk IIRC, so not an "on every save" issue).